Simplified/NYPLErrorLogger.swift

```diff
@@ -132,6 +136,66 @@ fileprivate let nullString = "null"
     }
   }
 
+  /// Broad areas providing some kind of operating context for error reporting.
+  /// These are meant to be related to the code base more than functionality,
+  /// (e.g. an error related to audiobooks may happen in different classes)
+  /// although the two things may obviously overlap.
+  enum Context: String {
```

Comment:
git is fine, it reported conflicts, I added the files because I saw it as files are missing and it's easier to remove from Xcode than edit Xcode project file manually. made it draft because fo this to test locally first

Reply:
That part is fixed now, I took the code from `develop` and changed one call that was using this Context enum